html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="/html-relative-links/about-me/about-me.html">About Me</a></li>
```

Comment:
These three urls should be updated, you don't want to path from the parent folder like this. note where the index.html file is in relation to the other folders and files. Reach out if you have questions!